spyder/utils/syntaxhighlighters.py

```diff
@@ -539,6 +541,7 @@ def highlight_match(self, text, match, key, value, offset,
                     oedata.def_type = OutlineExplorerData.CELL
                     def_name = get_code_cell_name(text)
                     oedata.def_name = def_name
+                    self.sig_new_cell.emit(oedata)

```

Comment:
Please add a comment why this is now needed.

Reply:
Done